### ISSUE TITLE
Internet Explorer 10

### ISSUE THREAD

**Issue by grav** [robinweser/react-layout-components#1]
I cannot get Flexbox working on IE 10. As I've understood it, Obscene Layout enables support on this browser, but if I make a simple example (a la `<Box><Item>foo</Item><Item>bar</Item></Box>`) or try the supplied `LayoutTest` component, IE 10 does not render correctly.

Did I misunderstand the scope of the project, or am I doing something wrong? :-)

I've attached a couple of screen shots to show the difference:

Chrome (Mac)
![screen shot 2015-06-09 at 21 28 09](https://cloud.githubusercontent.com/assets/119222/8067537/a54f9bea-0eee-11e5-8176-e478dcfd5592.png)

IE 10 (Win 8)
![screen shot 2015-06-09 at 21 28 22](https://cloud.githubusercontent.com/assets/119222/8067544/ac6ac7ec-0eee-11e5-875a-4bc8019c07de.png)

**Comment by robinweser** [comment]
Hey grav, thanks for your reply. As I just started coding on inline style modification last week I was kind of busy with information gathering and best ways to do things. Though I added all attributes as [Flexy Boxes](http://the-echoplex.net/flexyboxes/) and CanIUse mentioned there must have been something wrong.

Right now I was not able to test on IE at all, but I am just doing this right now, to fix this issue soon! Furthermore I'd like to inform you that I am currently working on Obscene Stylesheet to do all this stuff and let Obscene Layout only ship those nice components at last. 

Stay tuned,
Robin

**Comment by robinweser** [comment]
I just added a quickfix which was really a noobish one :>
I tested it with [netrenderer.de](netrenderer.de) which turned out to be a fast way to check pages again IE  engines. The static version I used to test is hosted on my [webpage](http://rofrischmann.de/obscene/flexbox/). Hope this fixes your issues
Thanks a lot for using! There is a overall flexbox support coming for Obscene Stylesheet soon, so you don't necessarily need `<Box>` & `<Item>` components to use flexbox layouts.
